src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((declaration) => declaration.trim())
+    .filter((declaration) => declaration)
+    .reduce((sourceObject, declaration) => {
+      const colonIndex = declaration.indexOf(':');
+
+      if (colonIndex !== -1) {
+        const property = declaration.slice(0, colonIndex).trim();
+        const value = declaration.slice(colonIndex + 1).trim();
+
+        if (property && value) {
+          sourceObject[property] = value;
```

Comment:
The current implementation assigns all values as strings. If the values are expected to be of different types (e.g., numbers, booleans), consider parsing them accordingly. For example, you could use `JSON.parse` if the values are JSON-compatible.